app/model/Developers2Filter.scala

```diff
@@ -32,4 +36,45 @@ object ApiContextVersion {
       case _ => throw new Exception("Invalid API context or version")
     }
   }
+}
+
+
+case object DeveloperStatusFilter {
+
+  sealed trait DeveloperStatusFilter {
+    def isMatch(user: User): Boolean
+
+    val value: String
+  }
+
+  case object UnverifiedStatus extends DeveloperStatusFilter {
+    val value = "UNVERIFIED"
+
+    override def isMatch(user: User): Boolean = user.verified match {
+      case None => false
+      case Some(verified) => !verified
+    }
+  }
+
+  case object VerifiedStatus extends DeveloperStatusFilter {
+    val value = "VERIFIED"
+
+    override def isMatch(user: User): Boolean = !UnverifiedStatus.isMatch(user)
```

Comment:
I think it would be clearer if we checked the flag on the VerifiedStatus e.g:
`
override def isMatch(user: User): Boolean = user.verified match {
      case None => false
      case Some(verified) => verified
    }
`

And then called this from UnverifiedStatus. e.g.
`
override def isMatch(user: User): Boolean = !VerifiedStatus.isMatch(user)
`